.teamcity/src/main/kotlin/configurations/FunctionalTest.kt

```diff
@@ -52,14 +52,6 @@ class FunctionalTest(
         extraSteps = extraBuildSteps,
         preSteps = preBuildSteps
     )
-
-    if (testCoverage.testType == TestType.soak || testTasks.contains("plugins:")) {
-        failureConditions {
-            // JavaExecDebugIntegrationTest.debug session fails without debugger might cause JVM crash
-            // Some soak tests produce OOM exceptions
-            javaCrash = false
```

Comment:
I think `javaCrash` = false needs to stay here to disable crash detection for all functional tests, right?